scripts/skills/perks/perk_rf_unstoppable.nut

```diff
@@ -30,85 +27,70 @@ this.perk_rf_unstoppable <- ::inherit("scripts/skills/skill", {
 		return this.m.Stacks == 0;
 	}
 
+	function getAPBonus()
+	{
+		return this.m.Stacks;
+	}
+
+	function getInitiativeBonus()
+	{
+		return this.m.Stacks * 10;
+	}
+
 	function getTooltip()
 	{
 		local tooltip = this.skill.getTooltip();
 
 		tooltip.push({
```

Comment:
This should be wrapped around by an `if (this.getAPBonus())` in case a submod changes how these values are generated.